src/System.Runtime.Extensions/ref/System.Runtime.Extensions.cs

```diff
@@ -893,6 +893,9 @@ public abstract partial class StringComparer : System.Collections.Generic.ICompa
         public static System.StringComparer InvariantCultureIgnoreCase { get { throw null; } }
         public static System.StringComparer Ordinal { get { throw null; } }
         public static System.StringComparer OrdinalIgnoreCase { get { throw null; } }
+#if netcoreapp
+        public static System.StringComparer FromComparison(System.StringComparison comparisonType) { throw null; }
+#endif
```

Comment:
Makes sense, though it was requested in https://github.com/dotnet/corefx/pull/14623#issuecomment-268071144.
Does the second part of the referenced comment still apply?